CHANGELOG.md

```diff
@@ -1,5 +1,103 @@
 # Changelog
 
+## 2021.6.99 (8 June 2021)
+
+### Enhancements
+
+1. Data Viewer Filter Rows must use explicit wildcards to search for substrings in string filters. For example, filtering by "stable" will not show the value "unstable" anymore, but filtering by "*stable" will show "stable" and "unstable".
+   ([#1142](https://github.com/Microsoft/vscode-jupyter/issues/1142))
+1. Sort variables by name and type in variable explorer.
+   ([#4585](https://github.com/Microsoft/vscode-jupyter/issues/4585))
+1. Limit languages dispalyed in the Cell language picker to languages supported by the kernel.
+   ([#5580](https://github.com/Microsoft/vscode-jupyter/issues/5580))
+1. Move native notebooks cell toolbar to the left by default.
+   ([#5605](https://github.com/Microsoft/vscode-jupyter/issues/5605))
+1. Display modal dialog box (so users don't miss this) when IPyKernel (or Jupyter) is missing (required to run Python in Interactive Window or Notebooks).
+   ([#5798](https://github.com/Microsoft/vscode-jupyter/issues/5798))
+1. Add support for [Virtual Workspaces](https://github.com/microsoft/vscode/wiki/Virtual-Workspaces).
+   ([#5803](https://github.com/Microsoft/vscode-jupyter/issues/5803))
+1. Losslessly compressed PNG images to save ~20KB.
```

Comment:
I believe this needs a thank you?

Reply:
it is there, line 84

Reply:
oh I see what you mean

Reply:
No an individual contributor thank you for this submission.

Reply:
I'll put it on the release branch, and it will be merged after we release